projects/igniteui-angular/src/lib/card/card-actions.component.html

```diff
@@ -1,10 +1,9 @@
 <div class="igx-card-actions__icons">
```

Comment:
Shouldn't we change the classes of the containers accordingly from `igx-card-actions__icons` to `igx-card-actions__end` and from `igx-card-actions__buttons` to `igx-card-actions__start`?